plugins/modules/pkgng.py

```diff
@@ -137,6 +144,13 @@
   community.general.pkgng:
     name: "*"
     state: latest
+
+# "use_globs" support added in 9.2.0
```

Comment:
This should say 9.3.0, but I would actually remove this comment since when something was added should be mainly documented in the option documentation.
```suggestion
```